### PR TITLE
add docs for  pause operation of cluster

### PR DESCRIPTION
**Pull request by dcy10000**

### CODE REVIEW

File: 2.Function/2.08_cluster.md [hunk 1]
```diff
@@ -160,20 +160,23 @@ dble启动时候会在online的目录下注册自己的信息，如果此时正
...
- 
+
 ##### G.1 pause流程
-1.申请分布式锁
-2.通知其他结点
-3.本结点停流量
-4.等待其他结点完成停流量 或者超时
-5.返回暂停成功或者失败
+1. 申请分布式锁 pause_node.lock
+3. 通知其他结点
```
Comment (yanhuqing666) [inline 2.Function/2.08_cluster.md:166]: 2

File: 2.Function/2.1_manager_cmd/2.1.14_pause_resume.md [hunk 2]
```diff
@@ -34,6 +35,6 @@ mysql> show @@pause;
 2 rows in set (0.15 sec)
 ```
 
-另外，商业版本支持集群操作，开源版本后续也将支持  
 
 
+注：商业版本和开源版本现已均支持 pause 操作。
```
Comment (yanhuqing666) [inline 2.Function/2.1_manager_cmd/2.1.14_pause_resume.md:40]: just delete it

File: 2.Function/2.08_cluster.md (hunk 1 above)
Comment (dcy10000) [reply 2.Function/2.08_cluster.md]: fixed

File: 2.Function/2.1_manager_cmd/2.1.14_pause_resume.md (hunk 2 above)
Comment (dcy10000) [reply 2.Function/2.1_manager_cmd/2.1.14_pause_resume.md]: fixed

**Review by yanhuqing666 — APPROVED** [review]
ok

**Pull request merged**